### ISSUE TITLE
Not sure where to put this as there is not GitHub Discussions

### ISSUE THREAD

**Issue by jgarte** [K1D77A/lunamech-matrix-api#7]
Hi,

Not sure where to put this as there is not GitHub Discussions but I wanted to ask you what you think of starting a project to create an alternative to [synapse](https://matrix.org/docs/projects/server/synapse) or dendrite, or the many other server implementations but in CL?

**Comment by jgarte** [comment]
ps

btw, i packaged this for guix:

https://issues.guix.gnu.org/62390

**Comment by K1D77A** [comment]
Yes it would be a cool project but it would also be a monumental project that I dont have time for.
